sections/platter-product-gallery.liquid

```diff
@@ -0,0 +1,118 @@
+<section class="container my-8 md:w-fit">
+  <header>
+    <h2 class="font-bold my-4 flex text-4xl justify-center md:text-6xl md:justify-start">{{ section.settings.heading }}</h2>
```

Comment:
This is the only section setting i've added apart from the necessary product card blocks.
But sure we could add more useful things as giving control on the product card image sizes, adding a "Quick buy" button, background color, etc...